web/src/lib/components/shared-components/search-bar/search-camera-section.svelte

```diff
@@ -26,13 +26,19 @@
       $type: SearchSuggestionType.CameraMake,
       model,
     });
+    if (filters.make && !makes.includes(filters.make)) {
+      filters.make = undefined;
+    }
```

Comment:
Why is this necessary?

Reply:
This just matches what was in the location filter file. If the current set filter isn't in the list of valid options it is discarded.

Reply:
Sure, but when can that actually happen?

Reply:
I think only by directly navigating with those params. Just seemed good to make them consistent, but maybe it isn't necessary

Reply:
It can be removed IMO, but it's fine if you prefer to keep it.